explore/src/main/scala/explore/HomeComponent.scala

```diff
@@ -40,28 +42,36 @@ object HomeComponent {
     ScalaComponent
       .builder[Unit]("Home")
       .initialState(0)
-      .renderPS { (_, _, i) =>
+      .renderPS { (_, _, _) =>
         <.div(
           ^.cls := "rgl-area",
           SizeMe() { s =>
-            <.div(
-              ResponsiveReactGridLayout(
-                s.width,
-                margin           = (5: JsNumber, 5: JsNumber),
-                containerPadding = (5: JsNumber, 5: JsNumber),
-                className        = "layout",
-                rowHeight        = 30,
-                draggableHandle  = ".tileTitle",
-                onLayoutChange   = (a, b) => Callback.log(a.toString) *> Callback.log(b.toString),
-                layouts          = layouts
-              )(
-                <.div(^.key := "tpe",
-                      ^.cls := "tile",
-                      Tile(Tile.Props("Conditions"), Conditions())),
-                <.div(^.key := "coords", ^.cls := "tile", Tile(Tile.Props("Coordinates"), Imag())),
-                <.div(^.key := "doc",    ^.cls := "tile", Tile(Tile.Props("Target Position"), Tpe(i)))
+            if(!js.isUndefined(s.width)) {
```

Comment:
Why not to use `unless` in suffix form?

Reply:
That causes a type mismatch. `unless` returns a `TagMod` and `SizeMe`'s function result expects a `VdomNode`.

Reply:
There must be a way around it, I have memories of doing this, maybe just wrap it on a div?

Reply:
That should work. I'll give it a go.

Reply:
Thing is, with suffix conditions, the block is evaluated before the condition and the error is still shown in the console. I will use `TagMod.unless(...)(...)`